server/aws-lsp-codewhisperer/src/language-server/chat/chatController.ts

```diff
@@ -170,8 +172,24 @@ export class ChatController implements ChatHandlers {
         this.#telemetryController.removeConversationId(params.tabId)
     }
 
-    onQuickAction(_params: QuickActionParams, _cancellationToken: CancellationToken): never {
-        throw new Error('Not implemented')
+    onQuickAction(params: QuickActionParams, _cancellationToken: CancellationToken) {
+        switch (params.quickAction) {
+            case QuickAction.Clear: {
+                const sessionResult = this.#chatSessionManagementService.getSession(params.tabId)
+
+                sessionResult.data?.clear()
```

Comment:
Do we need to do any telemetry related stuff here?

Once users clear the chat, it might mean that we should remove the conversation id for that tabid